### PR TITLE
certsync: update python resources

### PR DESCRIPTION
**Pull request by woodruffw**
This was done manually locally with `brew update-python-resources` to confirm the behavior of https://github.com/Homebrew/brew/pull/17724.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by woodruffw** [comment]
Audit is currently failing:

```
  * Stable resource "unicrypto": resource name should be `unicrypto-0-0-10-py3-none` to match the PyPI package name
Error: 1 problem in 1 formula detected.
```

I'll draft here and fix that.

**Comment by woodruffw** [comment]
https://github.com/Homebrew/brew/pull/17724 will include the resource auditor fix.

File: Formula/c/certsync.rb [hunk 1]
```diff
@@ -142,24 +142,23 @@ class Certsync < Formula
...
+    url "https://files.pythonhosted.org/packages/5a/c0/b7599d6e13fe0844b0cda01b9aaef9a0e87dbb10b06e4ee255d3fa1c79a2/tqdm-4.66.4.tar.gz"
+    sha256 "e4d936c9de8727928f3be6079590e97d9abfe8d39a590be678eb5919ffc186bb"
   end
 
-  # pypi artifact request, https://github.com/skelsec/unicrypto/issues/7
   resource "unicrypto" do
-    url "https://github.com/skelsec/unicrypto/archive/refs/tags/0.0.10.tar.gz"
-    sha256 "3daefcdf8e71f9300032393ad85d7f22fc08d07dc05cf7776019b9480bd8506a"
+    url "https://files.pythonhosted.org/packages/12/c5/41545c9f01fcf8651086459cce7b3795a8787edfb66362e1e05573d0e31d/unicrypto-0.0.10-py3-none-any.whl"
+    sha256 "77322c68cb6a7ef8ee762dcb0a824a491429f8939793e8a9d64f615baaf595b9"
```
Comment (woodruffw) [inline Formula/c/certsync.rb:151]: https://github.com/skelsec/unicrypto/issues/7 is still outstanding, but we're now using the universal wheel instead.

Comment (chenrui333) [reply Formula/c/certsync.rb:151]: can we use universal wheel now?

**Comment by chenrui333** [comment]
some syntax check failure

```
  certsync
    * Stable resource "unicrypto": resource name should be `unicrypto-0-0-10-py3-none` to match the PyPI package name
```

**Comment by woodruffw** [comment]
> some syntax check failure
> 
> 
> 
> ```
> 
>   certsync
> 
>     * Stable resource "unicrypto": resource name should be `unicrypto-0-0-10-py3-none` to match the PyPI package name
> 
> ```

Yep, the issue above covers that -- we can't use pure Python wheels until the audit change lands in a stable tag. So this is blocked until then 🙂

**Review by chenrui333 — COMMENTED** [review]
blocked atm

**Pull request closed without merging**